config-overrides.js

```diff
@@ -4,8 +4,10 @@ const { paths } = require('react-app-rewired');
 const { useBabelRc,override, addBundleVisualizer} = require('customize-cra')
 const { addReactRefresh } = require('customize-cra-react-refresh')
 const CompressionPlugin = require('compression-webpack-plugin')
+const Visualizer = require('webpack-visualizer-plugin');
```

Comment:
I'm reluctant to slow down the build to generate a graph we won't look at that often, can we put it behind a "process.env.visualiser" or some kind of conditional?